x-pack/legacy/plugins/epm/server/lib/elasticsearch/ilm/install.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CallESAsCurrentUser } from '../../../../server/lib/cluster_access';
+import { getPolicy } from './ilm';
+
+export async function installILMPolicy(name: string, callCluster: CallESAsCurrentUser) {
```

Comment:
Mind adding a return type here?

Reply:
I will as soon as I know what it should be. I returned `data` mainly for testing purpose but at the moment it is not used at all. Will follow up.